ai-gateway/concepts/models.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Models
+description: Understanding model naming and request formats in the AI Gateway
+---
+
+Models are the AI systems that process your requests. The AI Gateway supports flexible model naming and can route to the same model across different providers.
+
+## Specifying models
+
+You can specify which model to use in your requests. The gateway supports several formats depending on how specific you need to be.
+
+### Just the model name
+
+The simplest approach—use the model name directly:
+
+```json
+{
+  "model": "gpt-4o",
+  "messages": [{"role": "user", "content": "Hello"}]
+}
+```
+
+The gateway looks up `gpt-4o` in the [Model Catalog](/ai-gateway/reference/model-catalog) and automatically routes to the right provider (OpenAI in this case).
+
+### With a provider prefix
+
+Use `provider:model` when you want to explicitly choose a provider:
+
+```json
+{
+  "model": "anthropic:claude-3-5-sonnet-latest",
+  "messages": [{"role": "user", "content": "Hello"}]
+}
+```
+
+This is useful when:
+- A model name exists on multiple providers
+- You want to force a specific provider
+- You're using a custom provider you've configured
+
+### With model author
+
+Use the `author/model` format when you want to be specific about who authored the model:
+
+```json
+{
+  "model": "meta-llama/llama-3-70b",

```

Comment:
nit: these specific models might not make great examples given they're not from the model catalog